beeai/README.md

```diff
@@ -10,6 +10,138 @@ Three agents process tasks through Redis queues:
 - **Rebase Agent**: Updates packages to newer upstream versions
 - **Backport Agent**: Applies specific fixes/patches to packages
 
+### Entry Criteria
+
+Issues enter the system automatically via the Jira Issue Fetcher when they meet these criteria:
+- Belongs to the `Jotnar_1000_packages` filter
+- Status is "New"
+- No `jotnar_*` labels other than `jotnar_retry_needed`
+
+### Queue Transitions and Label Management
+
+The system uses Redis queues to route issues between agents and applies JIRA labels to track progress:
+
+**From Triage Agent:**
+- **REBASE decision** → `rebase_queue` + applies `jotnar_in_progress` label
+- **BACKPORT decision** → `backport_queue` + applies `jotnar_in_progress` label
```

Comment:
I'm still not totally sure if we need this, considering there will be the state change, or maybe at least change it to `jotnar_backport_in_progress` and `jotnar_rebase_in_progress`.